src/sentry/models/organizationonboardingtask.py

```diff
@@ -71,14 +71,46 @@ def record(self, organization_id, task, **kwargs):
         return False
 
 
-@region_silo_only_model
-class OrganizationOnboardingTask(Model):
+class AbstractOnboardingTask(Model):
```

Comment:
@roggenkemper can you also add a comment explaining we use in the sandbox? We should document all these kind of imports so someone doesn't later come in and wonder why this abstract class exists.